icu4c/source/common/loclikelysubtags.cpp

```diff
@@ -715,13 +715,28 @@ LSR LikelySubtags::maximize(StringPiece language, StringPiece script, StringPiec
             } else {
                 iter.resetToState64(state);
                 value = trieNext(iter, "", 0);
-                U_ASSERT(value > 0);
+                // For the case of und_Latn
+                if (value < 0) {
```

Comment:
Looks like we went from `U_ASSERT(value > 0)` to `if (value < 0)` here.  What if `value == 0`?  Does that case never happen?